### PR TITLE
Shadie conda-forge recipe

### PR DESCRIPTION
**Pull request by eaton-lab**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by elissasoroj** [comment]
I confirm that I am willing to maintain this package

**Comment by eaton-lab** [comment]
@conda-forge/help-python I think this is ready for review

File: recipes/shadie/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  run:
+    - python >=3.7
+    - slim >=4.2
+    - toytree >=3.0.1
+    - altair
+    - msprime
+    - tskit
+    - pyslim
+
+test:
```
[the remark this answers is not in the document]
Comment (synapticarbors) [reply recipes/shadie/meta.yaml:32]: You need to add the `requires` section with `pip` as is shown in the example I linked to in the previous comment.

**Comment by synapticarbors** [comment]
Thanks for adding the `pip check`. I see now that a few things are failing and may be misspecified upstream:

- In the `setup.cfg` it says that install requires `slim`, but that looks like a web framework on pypi and conflicts with the conda-forge `slim` which appears to be a "Forward-time population genetics simulator". My guess is that the `setup.cfg` should not include `slim` because python package config should work with tools like `pip` which are python-only package installers.
- `msprime` doesn't seem to be specified correctly in its feedstock. See https://github.com/conda-forge/msprime-feedstock/issues/64

I think everything looks fine with this PR and how it's specified. `msprime` is going to continue to be an issue. I'd just take a look at your upstream `setup.cfg` and make sure it's doing what you expect it to do vis-a-vis `slim`. Let's just  comment out the `pip check` for now.

File: recipes/shadie/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (eaton-lab) [reply recipes/shadie/meta.yaml:32]: Thank you. This has been addressed. Please see discussion below.

**Pull request merged**